src/components/updatemenus/draw.js

```diff
@@ -43,12 +43,12 @@ module.exports = function draw(gd) {
      *         <g item header />
      *         <text item header-arrow />
      *     <g header-group />
-     *         <g item header />
```

Comment:
AHHHH stupid vim plugin. It mangles html tags when you scroll incorrectly… restoring these.